litex/tools/litex_sim.py

```diff
@@ -62,6 +62,72 @@ class Platform(SimPlatform):
     def __init__(self):
         SimPlatform.__init__(self, "SIM", _io)
 
+# DFI PHY model settings ---------------------------------------------------------------------------
+
+sdr_phy_settings = {
+    "nphases": 1,
+    "rdphase": 0,
+    "wrphase": 0,
+    "rdcmdphase": 0,
+    "wrcmdphase": 0,
+    "cl": 2,
+    "read_latency": 4,
+    "write_latency": 0
+}
+
+sdram_module_rates = {
+    "SDR": "1:1",
+    "DDR": "1:2",
+    "LPDDR": "1:2",
+    "DDR2": "1:2",
+    "DDR3": "1:4",
+}
+
+def get_sdram_phy_settings(memtype, data_width, clk_freq):
+    nphases = 4 if memtype == "DDR3" else 2
+
+    # Settings taken from s7ddrphy
+    if memtype in ["DDR2", "DDR3"]:
+        tck                 = 2/(2*nphases*clk_freq)
+        cmd_latency         = 0
+        cl, cwl             = get_cl_cw(memtype, tck)
+        cl_sys_latency      = get_sys_latency(nphases, cl)
+        cwl                 = cwl + cmd_latency
+        cwl_sys_latency     = get_sys_latency(nphases, cwl)
+        rdcmdphase, rdphase = get_sys_phases(nphases, cl_sys_latency, cl)
+        wrcmdphase, wrphase = get_sys_phases(nphases, cwl_sys_latency, cwl)
+        read_latency        = 2 + cl_sys_latency + 2 + 3
+        write_latency       = cwl_sys_latency
+    # Settings taken from s6ddrphy
+    elif memtype in ["DDR", "LPDDR"]:
+        rdphase             = 0
+        wrphase             = 1
+        rdcmdphase          = 1
+        wrcmdphase          = 0
+        cl                  = 3
+        cwl                 = None
+        read_latency        = 5
+        write_latency       = 0
+
+    sdram_phy_settings = {
+        "nphases": nphases,
+        "rdphase": rdphase,
+        "wrphase": wrphase,
+        "rdcmdphase": rdcmdphase,
+        "wrcmdphase": wrcmdphase,
+        "cl": cl,
+        "cwl": cwl,
+        "read_latency": read_latency,
+        "write_latency": write_latency,
+    } if memtype != "SDR" else sdr_phy_settings
```

Comment:
The if can be removed when `sdr_phy_settings` will be generated in `get_sdram_phy_settings`.

Reply:
Removed as it is no longer needed